helix-lsp/src/lib.rs

```diff
@@ -181,6 +186,7 @@ pub mod util {
         match offset_encoding {
             OffsetEncoding::Utf8 => doc.try_byte_to_char(pos).ok(),
             OffsetEncoding::Utf16 => doc.try_utf16_cu_to_char(pos).ok(),
+            OffsetEncoding::Utf32 => Some(pos),
```

Comment:
The "TODO" above this `match` can probably be removed with this change, eh?

Reply:
Good point, forgot about that :+1: